### PR TITLE
docs: ng-conf 2020 added

### PR DESCRIPTION
**Pull request by owenmecham**
Added ng-conf 2020 marketing info to events and announcements

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [x] Other... Please describe: Marketing section of angular.io

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/marketing/announcements.json [hunk 1]
```diff
@@ -1,15 +1,15 @@
 [
   {
-    "startDate": "2019-04-02",
-    "endDate": "2019-04-20",
-    "message": "Join us for ng-conf<br/>May 1st-3rd, 2019",
+    "startDate": "2020-01-01",
```
Comment (gkalpak) [inline aio/content/marketing/announcements.json]: Did you mean "2019-**03**-01"? We usually show the bnotification for about a month. Three months might be too long.
(cc @StephenFluin)

**Review by gkalpak — CHANGES_REQUESTED** [review]
Thx! Left a couple of minor comments.
(BTW, are you involved with the conf organization. If not, we should probably check with the organizing team to make sure thay are happy with the content.)

File: aio/content/marketing/announcements.json (hunk 1 above)
Comment (owenmecham) [reply aio/content/marketing/announcements.json]: I will make an adjustment and then push my changes given those guidelines.

File: aio/content/marketing/announcements.json [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
+    "endDate": "2020-03-31",
+    "message": "Join us for ng-conf<br/>April 1st-3rd, 2020",
     "imageUrl": "generated/images/marketing/home/ng-conf.png",
```
[the remark this answers is not in the document]
Comment (owenmecham) [reply aio/content/marketing/announcements.json]: I can supply a new version of the logo for ng-conf 2020 if you can provide the dimensions that you require.

**Comment by owenmecham** [comment]
@gkalpak I am a new member of the ng-conf organizers. Aaron Frost asked me to submit this PR.

File: aio/content/marketing/announcements.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (StephenFluin) [reply aio/content/marketing/announcements.json]: You should replace and use the same dimensions as https://github.com/angular/angular/blob/master/aio/content/images/marketing/home/ng-conf.png (the current image) so it just keeps working.

**Review by StephenFluin — CHANGES_REQUESTED** [review]

**Comment by kapunahelewong** [comment]
Hey, @owenmecham! Thanks for this. Once you've addressed all the comments, I can help you get the PR through the rest of the process if you like. When you're ready, you'll want to squash and possibly rebase, and I have [notes](https://gist.github.com/kapunahelewong/567c30f7b353fd1c39bbf6d8343e2dca#how-to-rebase) on my own workflow for all of that if you'd like to refer to them. When you're ready or if you have any questions in the meantime, please tag me.

**Comment by kapunahelewong** [comment]
Hi @owenmecham,  a rebase should clear out all those extra commits that aren't part of this PR. You might like [these notes](https://gist.github.com/kapunahelewong/567c30f7b353fd1c39bbf6d8343e2dca#how-to-rebase) on my own workflow. If you need a hand, let me know and I can help you sort it out - I've done this before myself.🙂

**Comment by owenmecham** [comment]
@kapunahelewong I ran through the rebase steps from your process but it appears that this PR is in a bad place now. There are so few changes that it honestly would be faster for me to simply start again on a new branch and abandon this PR and start over.

**Comment by kapunahelewong** [comment]
Aw, ok, @owenmecham. Well, if you run into this again, maybe I can try my hand at it and see if I can sort it out from my end. Please feel free to tag me. 🙂

**Pull request closed without merging**